src/rust/lib/graph-editor/src/component/edge.rs

```diff
@@ -101,43 +245,68 @@ macro_rules! define_corner_end {($($color:tt)*) => {
 
                 let shadow_size = 10.px() + 1.px();
                 let n_radius = &shadow_size + 1.px() * dim.y();
-                let n_shape  = Rect((&shadow_size*2.0 + 2.px() * dim.x(),&n_radius*2.0)).corners_radius(n_radius);
+                let n_shape  = Rect(
+                (&shadow_size*2.0 + 2.px() * dim.x(),&n_radius*2.0)).corners_radius(n_radius);
```

Comment:
If you want to divide a line to multi-line expression, always use indentation. Now its hard to visually read it :)